pgmax/fg/groups.py

```diff
@@ -525,6 +526,14 @@ def __post_init__(self):
 
         object.__setattr__(self, "factor_type", type(self.factor))
 
+        if hasattr(self.factor, "configs"):
+            object.__setattr__(self, "factor_configs", self.factor.configs)
+
+        if hasattr(self.factor, "edge_states_offset"):
+            object.__setattr__(
+                self, "edge_states_offset", self.factor.edge_states_offset
+            )
```

Comment:
Because only EnumerationFactors have `configs` and only LogicalFactor have `edge_states_offset`.
However these arguments are needed for the SingleFactorGroup for inference